WhlgPortalWebsite/Models/ServiceManagerHomepageViewModel.cs

```diff
@@ -36,4 +37,10 @@ public AuthorityUserListing(User user)
             Manages = string.Join(", ", authorityNames);
         }
     }
+
+    public class NewDeliveryPartner
+    {
+        [EmailAddress(ErrorMessage = "Enter an email address in the correct format, like name@example.com")]
+        public string EmailAddress { get; set; } = string.Empty;
+    }
```

Comment:
it's not very common to nest view models like this. normally each view has its own non-nested class